Resources/Private/Javascript/Components/DefinitionList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import { classList } from '../Helper/utilities';
+
+const DefinitionList = ({ children, modifier }) => {
+  return (
+    <dl className={classList({
+      'c-definition-list': true,
+      ['c-definition-list--' + modifier]: modifier 
+    })}>
+      {children}
+    </dl>
+  );
+};
+
+export const DefinitionItem = ({ label, value, modifier }) => {
```

Comment:
A definition item can't or shouldn't be used without the list,. so i would make it a subitem. Before exporting the DefinitionList do something like DeifinitionList.DeifinitionItem = DefinitionItem. You don't need to import it then and can just use it as DefinitionList.DefinitionItem

Reply:
Will be fixed in later improvements